src/locale/en.ts

```diff
@@ -69,7 +69,9 @@ const translations = {
   },
   planList: {
     viewTitle: 'All Plans',
-    createPlan: 'Add a Plan',
+    createPlan: 'DODAJ NOWY PLAN',
+    copyPlan: 'SKOPIUJ ISTNIEJĄCY PLAN',
```

Comment:
I think it would be better to use style properties for uppercase `textTransform: 'uppercase'` instead of translation files

Reply:
And it's english translation file btw

Reply:
fixed